improver_tests/acceptance/acceptance.py

```diff
@@ -69,7 +69,7 @@ def run_cli(cli_name, verbose=True):
     def run_function(args):
         if not checksum_ignore():
             verify_checksums(args)
-        cli.main("improver", cli_name, *args, verbose=verbose)
+        return cli.main("improver", cli_name, *args, verbose=verbose)
```

Comment:
For next reviewer, this is needed to return the text output from the `--check-tree` option as this does not raise an exception. An exception could be raised as an alternative, if the tree is invalid, and then the test could be for an exception and this return would not be needed, but I think I prefer it as is.